waspc/cli/src/Wasp/Cli/Command.hs

```diff
@@ -17,8 +17,7 @@ newtype Command a = Command {_runCommand :: ExceptT CommandError IO a}
 
 runCommand :: Command a -> IO ()
 runCommand cmd = do
-  errorOrResult <- runExceptT $ _runCommand cmd
-  case errorOrResult of
+  runExceptT (_runCommand cmd) >>= \case
```

Comment:
Unrelated refactoring, but it is nice so I did it!

It is cool, with `\case` (which comes from LambdaCase extension) we can directly feed the result of monadic action into the case instead of having to extract it into a variable first, nice!
@matijaSos this is what I mentioned today to you.